.appveyor.yml

```
@@ -98,51 +98,62 @@ environment:
           VC: vc15
           OPCACHE: yes
           APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
-        - PHP_VER: 7.2
+        - PHP_VER: 7.2.34
           ARCH: x64
           TS: 1
           VC: vc15
           OPCACHE: yes
           APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
-        - PHP_VER: 7.2
+          LEGACY: yes
+        - PHP_VER: 7.2.34
           ARCH: x64
           TS: 0
           VC: vc15
           OPCACHE: no
           APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
-        - PHP_VER: 7.2
+          LEGACY: yes
+        - PHP_VER: 7.2.34
           ARCH: x86
           TS: 1
           VC: vc15
           OPCACHE: no
           APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
-        - PHP_VER: 7.2
+          LEGACY: yes
+        - PHP_VER: 7.2.34
           ARCH: x86
           TS: 0
           VC: vc15
           OPCACHE: yes
           APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+          LEGACY: yes
 
 build_script:
     ps: |
         $ErrorActionPreference = "Stop"
-        $gareleases = Invoke-WebRequest https://windows.php.net/downloads/releases/releases.json | ConvertFrom-Json
-        $qareleases = Invoke-WebRequest https://windows.php.net/downloads/qa/releases.json | ConvertFrom-Json
-        $garev = [regex]::split($gareleases.$env:PHP_VER.version, '[^\d]')[2]
-        $qarev = [regex]::split($qareleases.$env:PHP_VER.version, '[^\d]')[2]
-        if ($qarev -gt $garev) {
-            $env:PHP_VERSION = $qareleases.$env:PHP_VER.version
-            $env:PHP_RELEASE = 'QA'
+        if ('yes' -eq $env:LEGACY) {
```

You could get rid of the `LEGACY` env var by checking wether `PHP_VER` contains more than 1 dot:
```suggestion
        if (2 -le ([regex]::Matches($env:PHP_VER, '\.').count)) {
```